### ISSUE TITLE
Targetting a button via data-testid fails on web if the button also has aria-label set

### ISSUE THREAD

**Issue by hutchy2570** [mobile-dev-inc/maestro#3049]
### Is there an existing issue for this?

- [x] I have searched the existing issues and didn't find mine.

### Steps to reproduce

This issue only occurs when testing against Web.

Given a button:

```
<button aria-label="Some label" data-testid="some-test-id">
  <svg .../>
</button>
```

This test can't find the button:
```
- tapOn:
    id: some-test-id
```

### Actual results

Element cannot be found by maestro

Removing `aria-label` from the button results in maestro being able to find the element

### Expected results

Maestro will click the button which has `data-testid` of `some-test-id`

### About app

React native for web via expo

### About environment

Both locally (MacOS) and deployed to a cloud hosting service

### Logs

<details>
<summary>Logs</summary>

```
<!-- Replace this line with your logs. *DO NOT* remove the backticks! -->
```

</details>

### Maestro version

2.3.0

### How did you install Maestro?

install script (https://get.maestro.mobile.dev)

### Anything else?

I suspect it's related to how maestro web determine's the `resource-id`, as it appears to favours `ariaLabel` over `data-testid`.

https://github.com/mobile-dev-inc/Maestro/blob/46b646d7d3bfce990dc7b40a0c9eb3b04bad1f06/maestro-client/src/main/resources/maestro-web.js#L98

I was also unable to use `test: Aria label value` as the target due to this issue: https://github.com/mobile-dev-inc/Maestro/issues/2914
